### ISSUE TITLE
limit drop area

### ISSUE THREAD

**Issue by Tallyb** [SimpliField/angular-sf-dragndrop#2]
is there a way to limit the drop to certain values? e.g. in your example  - do not allow to drop on "Plunker"?

**Comment by nfroidure** [comment]
Do you mean having a property like Draggable for drop targets ? https://github.com/SimpliField/angular-sf-dragndrop#sf-draggable

It is not possible currently but having it working shouldn't be that difficult, feel free to submit a pull request, would happily merge one enabling this feature.

**Comment by Tallyb** [comment]
It should be a function that gets the dragged and dropped info (similar to sf-drag-end) and returns true to allow dropping or false to prevent dropping.

**Comment by nfroidure** [comment]
Made a simple poc of an sf-droppable property
http://plnkr.co/edit/4ynH09RVFHBFY5zw3bfe?p=preview

**Comment by Tallyb** [comment]
WOW! looks nice. does sf-droppable accepts a $scope function?

**Comment by nfroidure** [comment]
Yep, as it parses the attribute as an expression, but this naive test doesn't inject drag and drop variables like for other attributes. Not a hard thing to achieve though.

Feel free to PR those changes, otherwise, i'll have a look when possible.
